RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 12.8
 -----
-Add percentage bar to following Stats cards - Posts and Pages, Authors, Tags and Categories
+* Added Remote Preview support for Blog Posts
```

Comment:
IMHO "Add Remote Preview support for Drafts and Published posts with local changes" would be better.

Reply:
Considering some work to do with the open item related tickets, it definitely better describes what this PR brings. I just added also Scheduled and moved to 12.9. Changed here 4992a250e24b3658bf67e65282ba1ede9cd7a302.

Reply:
👍 We'll probably merge this in a feature branch (until we fix other known issues and have the corresponding fluxc branch merged in develop), so I guess we'll bump version number again later.